native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/listener/http2/Http2SourceConnectionHandler.java

```diff
@@ -152,6 +154,7 @@ public void onRstStreamRead(ChannelHandlerContext ctx, int streamId, long errorC
                 LOG.debug("RstStreamRead event in server frame listener. Stream id : {} Error code : {}", streamId,
                           errorCode);
             }
+            ctx.fireExceptionCaught(new Http2Exception(Http2Error.valueOf(errorCode), "Stream Closed"));
```

Comment:
```suggestion
            ctx.fireExceptionCaught(new Http2Exception(Http2Error.valueOf(errorCode), "Stream closed"));
```

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-http/pull/659/commits/2e53fcef70abec8eb45ea9c06b64cad8ffbdf0df